environment

```diff
@@ -78,6 +78,7 @@ EVENT_RELAY_AWS_USERNAME="dss-event-relay-${DSS_DEPLOYMENT_STAGE}"
 EVENT_RELAY_AWS_ACCESS_KEY_SECRETS_NAME="event-relay-user-aws-access-key"
 GOOGLE_APPLICATION_CREDENTIALS_SECRETS_NAME="gcp-credentials.json"
 GOOGLE_APPLICATION_SECRETS_SECRETS_NAME="application_secrets.json"
+DSS_TERRAFORM_BACKEND_BUCKET_TEMPLATE="org-humancellatlas-dss-config-861229788715"
```

Comment:
I don't think this should be hard-coded, right?

Reply:
I mean the account ID.

Reply:
That bucket deployed externally.

This seemed like the simplest way to parametrize with account id, but I'm all ears for alternate suggestions.

Reply:
Actually, I'm unsure why we need to include the account id here in the first place. Anyone deploying our product in another AWS account needs to customize each and every bucket name to avoid conflicts with the buckets in our account. So if we parameterize one bucket name with the account ID we need to parameterize all of them.

Lets talk tomorrow after stand-up.